pandas/io/parsers.py

```diff
@@ -574,7 +577,13 @@ def parser_f(filepath_or_buffer,
                  low_memory=_c_parser_defaults['low_memory'],
                  buffer_lines=None,
                  memory_map=False,
-                 float_precision=None):
+                 float_precision=None,
+
+                 # Basic auth (http/https) (username, password)
+                 auth=None,
+
+                 # skip verify self signed SSL certificates
```

Comment:
See my comment <a href="https://github.com/pandas-dev/pandas/pull/16910/files#r127319421">above</a>.  You should also be able to remove this comment.